types/bingmaps/Autosuggest.d.ts

```diff
@@ -0,0 +1,138 @@
+﻿/*
```

Comment:
Ok, This is actually a copy of another project so was hoping that it would be easy to update by simply copy/pasting the files over from the other project. I'll look at removing the license.